common/djangoapps/request_cache/__init__.py

```diff
@@ -73,11 +73,6 @@ def get_request_or_stub():
     request = crum.get_current_request()
 
     if request is None:
-        log.warning(
```

Comment:
@andy-armstrong so you think it is OK for this code to be hit in production? We could leave the warning here if we think future usages of this method by production code should be examined. With @ssemenova's other changes, we will be quieting the current way that this warning is triggered.